### PR TITLE
GPC simulator

### PR DESCRIPTION
**Pull request by nikita-paskal-dynatrace**

### CODE REVIEW

**Review by joaquinfilipic-dynatrace — APPROVED** [review]

File: gcp-simulator/run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+export DEV_URL='http://localhost:8080'
+
```
Comment (NematulloKozimov) [inline gcp-simulator/run.sh:3]: export GCP_METADATA_URL = 'http://localhost:8080/metadata.google.internal/computeMetadata/v1'
export GCP_CLOUD_RESOURCE_MANAGER_URL = 'http://localhost:8080/cloudresourcemanager.googleapis.com/v1'
export GCP_SERVICE_USAGE_URL = 'http://localhost:8080/serviceusage.googleapis.com/v1'
export GCP_MONITORING_URL = 'http://localhost:8080/monitoring.googleapis.com/v3'

Can we do this instead?

File: gcp-simulator/utils/tools.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+from utils.resources import Point, Metric
```
Comment (NematulloKozimov) [inline gcp-simulator/utils/tools.py:1]: Can we call this lib.py instead?

File: src/lib/configuration/config.py [hunk 3]
```diff
@@ -42,18 +42,32 @@ def release_tag():
 
 
 def gcp_metadata_url():
+    DEV_URL = os.environ.get("DEV_URL", None)
+    if DEV_URL:
+        return "http://localhost:8080/metadata.google.internal/computeMetadata/v1"
+
```
Comment (NematulloKozimov) [inline src/lib/configuration/config.py:48]: Please avoid making changes to dynatrace-gcp-monitor code. Same thing can be achieved by `export GCP_METADATA_URL = 'http://localhost:8080/metadata.google.internal/computeMetadata/v1'`

**Review by NematulloKozimov — CHANGES_REQUESTED** [review]

File: gcp-simulator/run.sh (hunk 1 above)
Comment (nikita-paskal-dynatrace) [reply gcp-simulator/run.sh:3]: Sure, makes more sense!

**Review by NematulloKozimov — APPROVED** [review]

**Review by NematulloKozimov — APPROVED** [review]

**Review by joaquinfilipic-dynatrace — APPROVED** [review]

**Pull request merged**